apps/web-giddh/src/app/multi-currency-reports/balance-sheet/components/balance-sheet-grid/balance-sheet-report-grid.component.ts

```diff
@@ -0,0 +1,230 @@
+    /** Stores local JSON data */
+    public localeData: any = {};
+    /** Stores common JSON data */
+    public commonLocaleData: any = {};
+    /** Indicates whether data should be hidden during search */
+    public hideData: boolean;
+    /** Indicates if expand all was toggled during search */
+    public isExpandToggledDuringSearch: boolean;
+
+    constructor(private cd: ChangeDetectorRef, private zone: NgZone) {
+        
+    }
+
+    /**
+     * Handles changes to the component's input properties
+     *
+     * @param {SimpleChanges} changes The changes object
+     * @memberof BalanceSheetReportGridComponent
+     */
+    public ngOnChanges(changes: SimpleChanges) {
+        if (changes.expandAll && !changes.expandAll.firstChange && changes.expandAll.currentValue !== changes.expandAll.previousValue) {
+            this.isExpandToggledDuringSearch = true;
+            if (this.bsData) {
+                this.zone.run(() => {
+                    if (this.bsData) {
+                        this.toggleVisibility(this.bsData.assets, changes.expandAll.currentValue);
+                        this.toggleVisibility(this.bsData.liabilities, changes.expandAll.currentValue);
+                        // always make first level visible ....
+                        if (this.bsData.liabilities) {
+                            each(this.bsData.liabilities, (childGroup: any) => {
+                                if (childGroup.isIncludedInSearch) {
+                                    childGroup.isVisible = true;
+                                    each(childGroup.accounts, (account: any) => {
+                                        if (account.isIncludedInSearch) {
+                                            account.isVisible = true;
+                                        }
+                                    });
+                                }
+                            });
+                        }
+                        if (this.bsData.assets) {
+                            each(this.bsData.assets, (childGroup: any) => {
+                                if (childGroup.isIncludedInSearch) {
+                                    childGroup.isVisible = true;
+                                    each(childGroup.accounts, (account: any) => {
+                                        if (account.isIncludedInSearch) {
+                                            account.isVisible = true;
+                                        }
+                                    });
+                                }
+                            });
+                        }
+
+                    }
+                    this.cd.detectChanges();
+                });
+            }
+        }
+    }
+
+    /**
+     * Initializes the component
+     *
+     * @memberof BalanceSheetReportGridComponent
+     */
+    public ngOnInit() {
+        this.lastSyncDate = dayjs(this.lastSyncDate, 'DD-MM-YYYY').format('DD MMMM YYYY');
```

Comment:
```suggestion
        this.lastSyncDate = dayjs(this.lastSyncDate, GIDDH_DATE_FORMAT).format('DD MMMM YYYY');
```

FOR this create new constant DD MMMM YYYY and use that

Reply:
Done